lute/static/js/lute.js

```diff
@@ -82,6 +82,8 @@ const _isUserUsingMobile = () => {
     return false;
   if (s == 'mobile')
     return true;
+  if (s == 'mobile-focus')
```

Comment:
As mentioned later, I'm not sure about "mobile-focus" being a "screen type", if that makes sense.  This might feel nitpicky, I'm not sure.  Just feels iffy to me at first glance.